.travis.yml

```diff
@@ -4,6 +4,12 @@ sudo: required
 services:
   - docker
 
+jobs:
+  include:
+   - arch: amd64
+   - arch: s390x
+     dist: bionic
```

Comment:
I don't think we should test one on trusty and one on bionic. Tests could error because of differences between those two.

Reply:
s390x build runs within an LXD container on Z-based infrastructure. LXD containers are with (Bionic), Xenial only.
Reference: https://docs.travis-ci.com/user/reference/overview/

Reply:
@SuperSandro2000 Could you please have a look ?

Reply:
I have nothing to say here but a good approach would be to update the other tests also to bionic.

Reply:
@SuperSandro2000 Added dist: bionic for `amd64` and `s390x`. Also set  GO111MODULE=on to resolve `l.LintPkg undefined (type *"github.com/golangci/lint-1".Linter has no field or method LintPkg)` error.
Ref: https://github.com/golangci/golangci-lint#go